eureka/S3_data_reduction/niriss.py

```diff
@@ -459,12 +473,119 @@ def wave_NIRISS(wavefile, meta):
     return meta
 
 
-def fit_bg(data, meta):
+def fit_bg(data, meta, n_iters=3, readnoise=5, sigclip=[4,2,3], isplots=0):
     """
     Subtracts background from non-spectral regions.
 
-    # want to create some background mask to pass in to 
-      background.fitbg2
+    Parameters
+    ----------
+    data : object
+    meta : object
+    n_iters : int, optional
```

Comment:
Can't this be derived from the length of sigClip?

Reply:
yeah, good point!

Reply:
I fixed this in the recent push